bucket/liteide.json

```diff
@@ -18,6 +18,6 @@
         "regex": "tag/x([\\d\\.]+)"
     },
     "autoupdate": {
-        "url": "https://downloads.sourceforge.net/project/liteide/x$version/liteidex$version.windows-qt5.9.5.zip"
+        "url": "https://github.com/visualfc/liteide/releases/download/x36.2/liteidex$version.windows-qt5.9.5.zip"
```

Comment:
```suggestion
        "url": "https://github.com/visualfc/liteide/releases/download/x$version/liteidex$version.windows-qt5.9.5.zip"
```

Reply:
Forgot to change it. :sweat_smile:. Thanks.